packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -198,18 +198,16 @@ export class StatsAnalyzer extends EventsScope {
     newMqa.intervalMetadata.peerReflexiveIP = this.statsResults.connectionType.local.ipAddress;
 
     // Adding peripheral information
-    newMqa.intervalMetadata.peripherals = [];
-
     newMqa.intervalMetadata.peripherals.push({information: _UNKNOWN_, name: MEDIA_DEVICES.SPEAKER});
     if (this.statsResults['audio-send']) {
       newMqa.intervalMetadata.peripherals.push({
-        information: this.statsResults['audio-send']?.trackLabel,
+        information: this.statsResults['audio-send'].trackLabel || _UNKNOWN_,
```

Comment:
We will need a jira to figure out why the trackLabel does not exist as this shouldn't happen, right?

Reply:
Looks like this label comes from wcme and has a chance to be undefined if there is no published stream on a transceiver:
https://sqbu-github.cisco.com/CPaaS/web-client-media-engine/blob/main/web-client-media-engine/src/transceiver-stats.ts#L72